### PR TITLE
create archetypes

### PR DESCRIPTION
**Pull request by chuckma**

### CODE REVIEW

**Comment by Salaboy** [comment]
@chuckma hi there, thanks a lot for this PR, can you please elaborate on what is the purpose for this PR? 

Notice that codecov is failing with the coverage and also you need to sign up for the CLA license.

File: tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	<groupId>${groupId}</groupId>
+	<artifactId>${artifactId}</artifactId>
+	<version>${version}</version>
+
+
+	<dependencies>
+		<dependency>
+			<groupId>org.activiti</groupId>
+			<artifactId>activiti-engine</artifactId>
+			<version>6.0.0</version>
```
Comment (Salaboy) [inline tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/pom.xml]: @chuckma if you are sending a PR to the branch develop, you shouldn't be using a different version

File: tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/src/main/java/DemoMain.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * @Author Lucas Ma
```
Comment (Salaboy) [inline tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/src/main/java/DemoMain.java]: @chuckma we don't use @Author tags

File: tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/src/main/java/DemoMain.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+ * <p>
+ * 启动类
+ */
+public class DemoMain {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(DemoMain.class);
+
+    public static void main(String[] args) throws ParseException {
+
+        LOGGER.info("启动程序");
```
Comment (Salaboy) [inline tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/src/main/java/DemoMain.java]: @chuckma comments and logs should be in english so other people can understand them.

File: tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/src/main/java/DemoMain.java [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+        // 处理流程任务
+        processTask(processEngine, processInstance);
+
+        LOGGER.info("结束程序");
+
+    }
+
+    private static void processTask(ProcessEngine processEngine, ProcessInstance processInstance) throws ParseException {
+        Scanner scanner = new Scanner(System.in);
```
Comment (Salaboy) [inline tooling/archetypes/activiti-archetype-unittest2/src/main/resources/archetype-resources/src/main/java/DemoMain.java]: @chuckma what is the purpose of this scanner here?

**Comment by salaboy** [comment]
closing due lack of feedback

**Pull request closed without merging**